config/job_conf.xml

```diff
@@ -96,11 +96,19 @@
         <destination id="papy-container" runner="k8s">
             <param id="docker_enabled">true</param>
         </destination>      
+	<destination id="isa2w4m-container" runner="k8s">
+	    <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
+            <param id="docker_owner_override">phnmnl</param>
+            <param id="docker_image_override">isa2w4m</param>
+            <param id="docker_tag_override">latest</param>
+            <param id="max_pod_retrials">3</param>
+            <param id="docker_enabled">true</param>
+        </destination>
 	<destination id="w4m-mtbls-dwnld-container" runner="k8s">
 	    <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
             <param id="docker_owner_override">phnmnl</param>
             <param id="docker_image_override">mtbls-dwnld</param>
-            <param id="docker_tag_override">v2.0.5_cv1.2.16</param>
+            <param id="docker_tag_override">latest</param>
```

Comment:
I would suggest that we use a fixed tag here, either a dev one, or even better, a release one if you are happy with the current state of the tool.

Reply:
Ok, I'll modify that and add also isa2w4m tool. I'm testing it now on minikube.